tests/e2e/cucumber/steps/app-files/resource.ts

```diff
@@ -34,11 +34,19 @@ When(
   }
 )
 
-Given(
-  '{string} downloads the following resource(s)',
-  async function (this: World, stepUser: string, stepTable: DataTable) {
+When(
+  /^"([^"]*)" downloads the following file(s)? using the (sidebar panel|batch action)$/,
```

Comment:
can we please use `resource` instead of `file` here, file isn't a good match if the user tries to download a folder.